app/controllers/course/admin/assessment_settings_controller.rb

```diff
@@ -0,0 +1,22 @@
+class Course::Admin::AssessmentSettingsController < Course::Admin::Controller
+  add_breadcrumb :index, :course_admin_assessments_path
+
+  def index
+  end
+
+  def update
+    if current_course.update_attributes(category_params)
+      redirect_to(course_admin_assessments_path, success: t('.success'))
+    else
+      redirect_to(course_admin_assessments_path, danger: t('.failure'))
```

Comment:
I'll change it to render, but there is no edit action: title and weight are edited via fields from the index and saved via nested attributes.

The reasoning was that when drag-and-drop is in place, an `edit` page would be overkill for only the title attribute. Plus we already have the capability for nested attributes here, so... What do you think?